### PR TITLE
Metasys style updates

### PR DESCRIPTION
**Pull request by nic-name**
- Put {{revision}} variable back
- Changed HTTP to HTTPS
- Removed Version specification rules

### CODE REVIEW

**Comment by mgwelch** [comment]
This renders #39 redundant. Addresses some of the comments from that PR

**Comment by DMahoney0** [comment]
@michaelgwelch @SheilaBonnar @nic-name I believe this PR is no longer needed.
I wanted to check with you before I closed it.

**Comment by mgwelch** [comment]
> @michaelgwelch @SheilaBonnar @nic-name I believe this PR is no longer needed.
> I wanted to check with you before I closed it.

I'm not sure. These are rather recent so I would assume these are changes that Nicola-Marie was making to correct style. However, I observe two odd things:

1. The list of error codes is larger in the newer version. I wouldn't expect style changes to add error codes, unless we specifically noted we were missing some and asked her to add them
2. The last paragraph talks about how redirects may be removed in the future. This is obviously an out of data paragraph that perhaps we never removed when we published 10.1.

**Comment by DMahoney0** [comment]
@SheilaBonnar let me know if you need help updating or rebasing this PR.

File: _templates/header.apib [hunk 1]
```diff
@@ -25,63 +25,66 @@ This documentation was last updated on {{revision}}.
...
+|Record not authorized          |403 (Forbidden)             |You are not authorized to view data matching the provided identifier                                       |
+|Identifier not found           |404 (Not Found)             |The identifier you provided does not match any known data                                                  |
+|The Resource already exists    |409 (Conflict)              |The resource already exists                                                                                |
+|Internal Server Error          |500 (Internal Server Error) |An unexpected error occurs                                                                                 |
+|The device is not supported    |501 (Not Implemented)       |The server does not support the functionality it requires to fulfill the request                           |
+|The device is offline          |503 (Service Unavailable)   |The device is currently offline and unable to serve the request                                            |
 
 ## Redirects
```
Comment (DMahoney0) [inline _templates/header.apib]: Redirects can be removed. They were in 10.0 and got removed in 10.1.

**Comment by SheilaBonnar** [comment]
@DMahoney0 , please review previous commit, UPDATE HEADER.APIB

File: _templates/header.apib [hunk 2]
```diff
@@ -25,63 +25,75 @@ This documentation was last updated on {{revision}}.
...
-* If the version specified is invalid (i.e. does not represent a known version of the API), a 404 (not found) error status will be returned.
+Because version specification is optional and the user has a choice of options, note the following system rules:
+* If the consumer does not specify a version, the lowest supported version opens.
+* If the consumer specifies a version in the URL and the `Accept` header, the system uses the version specified in the URL. The system ignores the version specified in the header.
+* If the consumer specifies an invalid version, a 404 (not found) error status appears.
 
 #### Version Response Behavior
 
-The `Content-Type` header of the response will always contain (in part) the version of the API that was used to serve the request. This allows the consumer to identify which version of the API was utilized.
+To enable the consumer to identify, which version of the API is active, the `Content-Type` header of the response always contains in part, the version of the API that you used to server the request.
```
Comment (SheilaBonnar) [inline _templates/header.apib:55]: Change server to serve

**Comment by DMahoney0** [comment]
@SheilaBonnar the number of conflicts in header.apib is making it hard to merge. I will remake this PR.

**Comment by SheilaBonnar** [comment]
Thanks

**Comment by DMahoney0** [comment]
redone in https://github.com/metasys-server/api-docs/pull/55 due conflicts

**Pull request closed without merging**